examples/create-react-app/src/app.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import Pdf from './pdf';
+
+export default function App () {
+  return (
+    <main>
+      <h1>Simple pdf render example</h1>
```

Comment:
`PDF.js rendering example for React` also here maybe?

Reply:
For sure! Already changed it